internal/stage_clone_repository.go

```
@@ -90,6 +92,46 @@ func testCloneRepository(harness *test_case_harness.TestCaseHarness) error {
+	copyCmd := exec.Command("sh", "-c", command)
+	copyCmd.Stdout = os.Stdout
+	copyCmd.Stderr = os.Stderr
+	if err := copyCmd.Run(); err != nil {
+		return fmt.Errorf("CodeCrafters Internal Error: mv1 failed: %w", err)
+	}
+	logger.Debugf("mv-ed git to temp directory: %s", tmpGitDir)
+
+	logger.Infof("$ git clone %s %s", testRepo.url, "test_dir")
+
+	//////// which git
+	command = fmt.Sprint("which git")
```

golangci-lint (1.64.8)

S1039: unnecessary use of fmt.Sprint

(gosimple)